### DISCUSSION TITLE
How do I add a second keyboard layout? The + button in regolith settings does not work

### DISCUSSION THREAD

**Discussion by JohannesPertl** [regolith-linux/regolith-desktop#680] [Q&A]
Adding a new keyboard layout via the regolith settings menu does not work, is there a workaround?

**Comment by JohannesPertl** [comment]
The normal settings are apparently still there (opened via ilia) and work just fine

**Comment by kgilmer** [comment]
To make sure I understand what you mean, `regolith-control-center` does not allow in UI for additional keyboard layout (I just tried and it worked for me), or that you can set it but it has no effect?

**Reply by JohannesPertl** [reply]
The first one, but it seems to be a problem of my specific install. I just installed Regolith on another device and there it works just fine. I'll try re-installing on the problematic device

**Reply by kgilmer** [reply]
Sure, LMK what you find either way :)
